packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -115,33 +112,96 @@ function AnimalInventory({ isCompactSideMenu }: AnimalInventoryProps) {
     [t, isMobile],
   );
 
+  // TODO: Adjust once LF-4093 & 4094 are merged
+  const filteredInventory = inventory.slice();
```

Comment:
I'm getting an error that `filteredInventory` is alredy declared so I think this line needs to be removed now that the filters PR got merged

Reply:
I'm sorry, I totally forgot about this... It has been removed, thank you!!